app/containers/ActionSheet/ActionSheet.tsx

```diff
@@ -125,6 +125,11 @@ const ActionSheet = React.memo(
 		// when is android tablet and the input text is focused
 		const androidTablet: any = isTablet && isLandscape && !isIOS ? { android_keyboardInputMode: 'adjustResize' } : {};
 
+		const handleOnChange = (index: number) => {
```

Comment:
Your type is accurate, but you should do `BottomSheetProps['onChange']`.
Please use this strategy whenever TS can't infer types automatically from a lib outside our app.